.pre-commit-config.yaml

```diff
@@ -0,0 +1,15 @@
+repos:
+-   repo: https://gitlab.com/pycqa/flake8
+    rev: 3.7.7
+    hooks:
+    -   id: flake8
+        args: [--max-line-length=88]
+-   repo: https://github.com/python/black
+    rev: 19.3b0
+    hooks:
+    -   id: black
+default_language_version:
+    python: python3.6
+    python: python3.7
```

Comment:
Hmm, I'm not sure what would be the semantics here as `default_language_version` is a map and you supply two values for the same key. Nevertheless, let's stick to 3.6 since it's the least version we support.